base/src/main/AndroidManifest.xml

```diff
@@ -15,22 +15,20 @@
   ~ limitations under the License.
```

Comment:
Change copyright year to 2020?

Reply:
If I'm just updating a file, I don't need to change the year. I only set that based on the year it was created. :)